src/internal/connector/exchange/exchange_data_collection.go

```diff
@@ -115,19 +119,20 @@ func (col *Collection) populateByOptionIdentifier(
 ) {
 	var (
 		errs       error
-		success    int
+		success    int64
```

Comment:
I had to switch it to either `int32` or `int64` as I needed to do an [atomic increment](https://github.com/alcionai/corso/blob/b1b351b449f046a0282d88b2989403505b59313d/src/internal/connector/exchange/exchange_data_collection.go#L192). And since `int` will be `int64` on most platforms we will be deploying, I thought I would go with `int64`.